python/libcuvs/pyproject.toml

```diff
@@ -105,4 +105,4 @@ select = [
 ]
 
 # detect when package size grows significantly
-max_allowed_size_compressed = '1.1G'
+max_allowed_size_compressed = '1.2G'
```

Comment:
Hi @jameslamb , for conservative consideration, I increase it to 1.2GB. Thanks!

Reply:
ok with me